docs/notebooks/synthetic_data_uber_differential_privacy.ipynb

```diff
@@ -0,0 +1,1462 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "accelerator": "GPU",
+    "colab": {
+      "name": "synthetic-data-uber-differential-privacy.ipynb",
+      "provenance": [],
+      "collapsed_sections": [],
+      "toc_visible": true
+    },
+    "kernelspec": {
+      "display_name": "Python 3",
+      "language": "python",
+      "name": "python3"
+    },
+    "language_info": {
+      "codemirror_mode": {
+        "name": "ipython",
+        "version": 3
+      },
+      "file_extension": ".py",
+      "mimetype": "text/x-python",
+      "name": "python",
+      "nbconvert_exporter": "python",
+      "pygments_lexer": "ipython3",
+      "version": "3.6.9"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "UTRxpSlaczHY"
+      },
+      "source": [
+        "# A differentially private, synthetic ride-share dataset\n",
+        "\n",
+        "This blueprint utilizes Gretel's premium SDKs to create a synthetic version of your own data. Our SDKs create automatic data validators to help ensure the data generated has the same semantics as the source data. Additionally, the SDKs do autmoatic header clustering to help maintain statistical relations between columns."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "VEM6kjRsczHd"
+      },
+      "source": [
+        "%%capture\n",
+        "\n",
+        "!pip install -U gretel-client gretel-synthetics pandas"
```

Comment:
gretel-client includes gretel-synthetics, so we can remove that from the installs.